pkg/server/container_create.go

```diff
@@ -118,7 +119,15 @@ func (c *criContainerdService) CreateContainer(ctx context.Context, r *runtime.C
 		}
 	}()
 
-	opts = append(opts, containerd.WithSpec(spec), containerd.WithRuntime(defaultRuntime))
+	b,err := meta.Encode()
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert sandbox metadata: %+v, %v", meta, err)
+	}
+	labels[containerMetadataLabel] = string(b)
```

Comment:
Even we add new label, I prefer group the logic together, so that we know what labels are added at a glance.